doc/content/plugins/feeds.rst

```diff
@@ -75,6 +84,22 @@ Usage
             # this callback is used to generate the 'updated' flag
             # of every feed entry
             'updated':
-                lambda content: content['date'].strftime('%Y-%m-%d %H:%M:%S+01:00')
+                lambda content: content['date'].strftime('%Y-%m-%d %H:%M:%S+01:00'),
+
+            # this callback is used to generate the 'published' flag
+            # of every feed entry
+            'published':
+                lambda content: content['date'].strftime('%Y-%m-%d %H:%M:%S+01:00'),
         },
     ]
+
+
+For the feed type `podcast` every content returned by the `contents` -filter must contain a header
+field in the following format:
+
+.. code-block:: python
+
+    podcast:
+      url: http://url.to/file.mp3
+      size: 123456789   # in bytes
+      type: audio/mpeg  # optional, defaults to audio/mpeg
```

Comment:
Having to manually put the size here is a bit annoying, but if the spec requires it there is not much we can do about it
(Without requiring access to the file at build time to calculate the size, which is also not a great solution).
So this looks fine to me.
I like the option to manually set the mime type.

Reply:
> (Without requiring access to the file at build time to calculate the size, which is also not a great solution).

This would be OK, if we could rely on the file being in the `content/` directory of the project we are building. But since gigabytes of MP3s aren't going great with git I don't want to rely on this. 
Also: always doing a `HEAD` request for all reference files during build does also not look like the right way.